pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -13,20 +13,21 @@
 , requests
 , smbprotocol
 , tqdm
+, nix-update-script
 }:
 
 buildPythonPackage rec {
   pname = "fsspec";
-  version = "2022.8.2";
+  version = "2022.10.0";
```

Comment:
It's usually required to update [gcsfs](https://github.com/fsspec/gcsfs/blob/2022.10.0/requirements.txt#L6), [s3fs](https://github.com/fsspec/s3fs/blob/2022.10.0/requirements.txt#L2) and perhaps others modules in the process.

Reply:
Updated gcsfs and s3fs to 2022.10.0